package.json

```diff
@@ -4,7 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "node ./lib/scrape.js && node ./lib/generate.js"
+    "scrape": "node ./lib/scrape.js",
+    "generate": "node ./lib/generate.js",
+    "gather": "node ./lib/gather.js",
+    "clean": "rm -rf out && mkdir -p out && touch out/.keep",
```

Comment:
Remove the contents instead of the folder
i.e. `rm -rf out/*` 
and if you do that yo don't need to recreate the `.keep` file